quarkus-integration/quarkus/deployment/src/test/java/ai/timefold/solver/quarkus/TimefoldProcessorSupplierShadowSolutionSimpleSolveTest.java

```diff
@@ -10,9 +10,9 @@
 import jakarta.inject.Inject;
 
 import ai.timefold.solver.core.api.solver.SolverManager;
-import ai.timefold.solver.quarkus.testdomain.suppliervariable.constraints.TestdataQuarkusSupplierVariableConstraintProvider;
-import ai.timefold.solver.quarkus.testdomain.suppliervariable.domain.TestdataQuarkusSupplierVariableEntity;
-import ai.timefold.solver.quarkus.testdomain.suppliervariable.domain.TestdataQuarkusSupplierVariableSolution;
+import ai.timefold.solver.quarkus.testdomain.suppliervariable.simple.constraints.TestdataQuarkusSupplierVariableSimpleConstraintProvider;
+import ai.timefold.solver.quarkus.testdomain.suppliervariable.simple.domain.TestdataQuarkusSupplierVariableSimpleEntity;
+import ai.timefold.solver.quarkus.testdomain.suppliervariable.simple.domain.TestdataQuarkusSupplierVariableSimpleSolution;
```

Comment:
Considering that the FQN is already very long, do we need to repeat the `domain` there?

Reply:
That how all the other `testdomain` classes does it.

Reply:
Not anymore. :-) We changed it in `core`, we shortened the FQNs significantly.
(They started giving trouble on Windows, long paths.)